src/main/java/seedu/address/MainApp.java

```diff
@@ -85,7 +85,7 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
         ReadOnlyAddressBook addressBook;
         try {
             addressBookOptional = storage.readAddressBook();
-            if (!addressBookOptional.isPresent()) {
+            if (addressBookOptional.isEmpty()) {
```

Comment:
Lol, this is umm.. subtle. : )

Reply:
The IDE gives me some lints, so i just corrected it according to the IDE